nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -2393,19 +2395,53 @@ When the DFM makes changes to the dataflow, the node that receives the request t
 nodes and waits for each node to respond, indicating that it has made the change on its local flow.
 
 
-*Dealing with Disconnected Nodes* +
+=== Managing Nodes
+
+==== Disconnect Nodes
+
+A DFM may manually disconnect a node from the cluster. A node may also become disconnected for other reasons, such as due to a lack of heartbeat. The Cluster Coordinator will show a bulletin on the User Interface when a node is disconnected. The DFM will not be able to make any changes to the dataflow until the issue of the disconnected node is resolved. The DFM or the Administrator will need to troubleshoot the issue with the node and resolve it before any new changes can be made to the dataflow. However, it is worth noting that just because a node is disconnected does not mean that it is not working.  This may happen for a few reasons, for example when the node is unable to communicate with the Cluster Coordinator due to network problems.
+
+To manually disconnect a node, select the "Disconnect" icon (image:iconDisconnect.png["Disconnect Icon"]) from the node's row.
+
+image::disconnected-node-cluster-mgt.png["Disconnected Node in Cluster Management UI"]
+
+A disconnected node can be connected (image:iconConnect.png["Connect Icon"]), offloaded (image:iconOffload.png["Offload Icon"]) or deleted (image:iconDelete.png["Delete Icon"]).
```

Comment:
For a node that’s disconnected due to lack of heartbeat, it can't offload in all disconnected scenarios.  If it gets disconnected because the node died, obviously it won’t be able to be offloaded, but if the operation can’t reach the node to start the offloading, it won’t be able to start it and the UI should reflect that error.  If it is disconnected due to firewall issues, that might effect offloading as well, if the load balancing port is also blocked by the firewall.

Reply:
Good points.  My intent was to present the available choices provided by the UI for a disconnected node.  Perhaps we can add a note to the end of this section that provides more of this error handling/troubleshooting information.

NOTE: Not all nodes in a "Disconnected" state can be offloaded.  If the node is disconnected because the node died/lack of heartbeat, the offload operation will be unable to reach the node to start the offloading.  Additionally, offloading may also be interrupted or prevented due to firewall issues (e.g., if the load balancing port is blocked).

Reply:
I misspoke on the load balance port being blocked.  That would prevent other nodes from sending flowfiles through load balancing to that particular node.  If the disconnected node is unreachable, it would not be able to receive the offload request.  I agree with moving the error scenarios to the end of the section, or to a separate error handling/troubleshooting section.

Reply:
OK.  How about this then for the note:

NOTE: Not all nodes in a "Disconnected" state can be offloaded. If the node is disconnected because the node died/lack of heartbeat, the offload operation will be unable to reach the node to start the offloading. Additionally, offloading may also be interrupted or prevented due to firewall issues.

Reply:
Maybe it would be better to be less explicit:

NOTE: Not all nodes in a "Disconnected" state can be offloaded. If the node is disconnected and unreachable, the offload request can not be received by the node to start the offloading. Additionally, offloading may be interrupted or prevented due to firewall rules.

Reply:
I like your version.  Let's run with that.

Reply:
+1